homeassistant/components/homewizard/sensor.py

```diff
@@ -108,13 +108,28 @@ class HomeWizardSensorEntityDescription(SensorEntityDescription):
         has_fn=lambda data: data.total_energy_import_kwh is not None,
         value_fn=lambda data: data.total_energy_import_kwh,
     ),
+    HomeWizardSensorEntityDescription(
+        key="total_power_import_t1_kwh",
+        translation_key="total_energy_import_kwh",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        has_fn=lambda data: (  # HWE-SKT provides tarif 1 only
+            data.total_energy_import_t1_kwh is not None
+            and data.total_energy_import_kwh is None
+        ),
+        value_fn=lambda data: data.total_energy_import_t1_kwh,
+    ),
     HomeWizardSensorEntityDescription(
         key="total_power_import_t1_kwh",
         translation_key="total_energy_import_t1_kwh",
         native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
-        has_fn=lambda data: data.total_energy_import_t1_kwh is not None,
+        has_fn=lambda data: (  # SDM230/630 provides both total and tarif 1: duplicate.
+            data.total_energy_import_t1_kwh is not None
+            and data.total_energy_export_t2_kwh is not None
+        ),
```

Comment:
```suggestion
        has_fn=lambda data: (  # HWE-SKT/SDM230/630 provides both total and tarif 1: duplicate.
            data.total_energy_import_t1_kwh is not None
            and data.total_energy_export_t2_kwh is not None
        ),
```

Reply:
Fixed it during the rebase onto #104537 (as it caused merge conflicts/test conflicts)